src/main/kotlin/org/rust/ide/annotator/RsErrorAnnotator.kt

```diff
@@ -457,6 +458,11 @@ class RsErrorAnnotator : Annotator, HighlightRangeExtension {
         holder.createErrorAnnotation(ret, "`return;` in a function whose return type is not `()` [E0069]")
     }
 
+    private fun checkExternCrate(holder: AnnotationHolder, el: RsExternCrateItem) {
+        if (el.reference.resolve() != null) return
+        holder.createErrorAnnotation(el.textRange, "Unknown crate '" + el.identifier.text + "' [E0463]")
```

Comment:
Let's change this message to match what rustc emits: 

```
Can't find crate for `${el.identifier.text}` [E0463]
```

as we do in all such annotations.